base_location/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Nicolas Bessi, Camptocamp SA
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Location management (aka Better ZIP)',
+    'version': '12.0.1.0.0',
+    'depends': [
+        'base_address_city'
+    ],
+    'author': "Camptocamp,"
+              "ACYSOS S.L.,"
+              "Alejandro Santana,"
+              "Tecnativa,"
+              "AdaptiveCity,"
+              "Odoo Community Association (OCA)",
+    'license': "AGPL-3",
+    'summary': '''Enhanced zip/npa management system''',
+    'website': 'http://www.camptocamp.com',
+    'data': ['views/better_zip_view.xml',
```

Comment:
Follow this sequence https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L27